test/BUILD.gn

```diff
@@ -39,436 +39,496 @@ static_library("brave_test_support_unit") {
   ]
 
   public_deps = [
-    "//chrome/test:test_support",
     "//base",
     "//chrome:resources",
     "//chrome:strings",
     "//chrome/browser",
     "//chrome/common",
+    "//chrome/test:test_support",
   ]
 
   deps = [
-    "//components/gcm_driver:gcm_driver",
+    "//brave/browser/profiles:profiles",
+    "//brave/common:pref_names",
     "//components/gcm_driver:gcm_buildflags",
+    "//components/gcm_driver:gcm_driver",
```

Comment:
no need to specify exact target after :, just  `"//components/gcm_driver"`

Reply:
Done.